app/jobs/course/material/zip_download_job.rb

```diff
@@ -11,7 +11,7 @@ class Course::Material::ZipDownloadJob < ApplicationJob
   #   is useful when you don't want to use the name of the folder as the zip filename (such as the
   #   root folder).
   def perform_tracked(folder, materials, filename = folder.name)
-    zip_file = Course::Material::ZipDownloadService.download_and_zip(folder, materials)
+    zip_file = Course::Material::ZipDownloadService.download_and_zip(materials, folder)
```

Comment:
I think "download materials" is more intuitive, while folder is just a reference there( it actually can be calculated), that's why I swapped them.. 

What do you think  ?